src/modules/cards/cards.module.scss

```diff
@@ -1,24 +1,25 @@
-@import 'src/assets/stylesheets/base_styles/typography';
-@import 'src/assets/stylesheets/base_styles/colors';
-@import 'src/assets/stylesheets/utilities/functions';
-@import 'src/assets/stylesheets/utilities/mixins';
+@import 'src/assets/stylesheets/global';
```

Comment:
@jmcolella Kind of.. but do we want to do it explicitly, or do we want it to simply be part of our post-css pipeline?